src/components/Dropdown/Dropdown.tsx

```diff
@@ -159,7 +159,7 @@ export const Dropdown = ({
             id={`select-${componentId}`}
             aria-labelledby={label && `label-${componentId}`}
             aria-describedby={helperText && `helper-${componentId}`}
-            data-testid='dropdown'
+            data-testid="dropdown"
```

Comment:
Otatko mainista viimeisimmät tavarat, niin ei tulis näitä tarpeettomia muutoksia. Mainissa on tuoreet konffit, niin tätä muutosta ei pitäs tulla.

Reply:
Juu

Reply:
korjattu